base/Enums.jl

```diff
@@ -93,6 +93,15 @@ macro enum(T,syms...)
             end
         end
         Base.show(io::IO,x::$(esc(typename))) = print(io, x, "::", $(esc(typename)))
+        function Base.show(io::IO,x::Type{$(esc(T))})
```

Comment:
Seems that `T` is always the same with `typename` ?

Also `x` can be removed..